http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpWsmDao.scala

```diff
@@ -185,11 +186,20 @@ class HttpWsmDao[F[_]](httpClient: Client[F], config: HttpWsmDaoConfig)(implicit
                                                          false
       )
       aksSubnetName <- getLandingZoneResourceName(groupedLzResources, "DeployedSubnet", AKS_NODE_POOL_SUBNET, false)
-      postgresName <- getLandingZoneResourceName(groupedLzResources,
+      postgresResource <- getLandingZoneResource(groupedLzResources,
                                                  "Microsoft.DBforPostgreSQL/flexibleServers",
-                                                 SHARED_RESOURCE,
-                                                 false
+                                                 SHARED_RESOURCE
       ).attempt
+      postgresServer = postgresResource.toOption.flatMap { resource =>
+        getLandingZoneResourceName(resource, false).map { pgName =>
```

Comment:
In lieu of named parameters (does Scala have something like this?) I think the `false` arg could use an inline comment to help its readability, eg: `/* useParent= */`.  Otherwise I have to crack open the method definition to understand what `false` is referring to.

Reply:
In Scala you don't have to comment, you can name params in function calls directly:
```
getLandingZoneResourceName(resource, useParent = false)
```
It's often used when there are defaults in the method definitions, so you might not be passing the full parameter list each time.